step/download_external_resources.sh

```diff
@@ -3,9 +3,9 @@
 download="$utils/download.sh"
 extract="$utils/extract.sh"
 
-jdk_url='https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u312-b07/OpenJDK8U-jre_x64_windows_hotspot_8u312b07.zip'
+jdk_url='https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u412-b08/OpenJDK8U-jre_x86-32_windows_hotspot_8u412b08.zip'
```

Comment:
JDK is finicky, I tested a few different builds and tools and found this version to be the least problematic. It's better to do this update in a separate PR, with a bit more testing and discussion on what it breaks and fixes.

Reply:
sounds fair reverted the change!